### PR TITLE
mod_wsgi: Update to 4.9.4

### PR DESCRIPTION
**Pull request by ghost**
Update Apache Mod wsgi to the latest version 4.9.4.
The version we currently offer is listed as vulnerable in Repology.

### CODE REVIEW

File: components/web/apache2-modules/mod_wsgi/mod_wsgi-27.p5m [hunk 1]
```diff
@@ -21,10 +21,12 @@
...
 #
 # Copyright (c) 2012, 2013, Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2022 Niklas Poslovski
 #
 
 set name=pkg.fmri \
     value=pkg:/web/server/apache-24/module/apache-wsgi-27@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.human-version value=$(HUMAN_VERSION)
 set name=pkg.summary \
     value="Python 2.7 mod_wsgi plugin for Apache Web Server v2.4"
```
Comment (mtelka) [inline components/web/apache2-modules/mod_wsgi/mod_wsgi-27.p5m]: Python 2.7 is no longer supported.  I suggest to obsolete `apache-wsgi-27`.

File: components/web/apache2-modules/mod_wsgi/mod_wsgi-35.p5m [hunk 2]
```diff
@@ -21,10 +21,12 @@
 
 #
 # Copyright (c) 2012, 2013, Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2022 Niklas Poslovski
 #
 
 set name=pkg.fmri \
     value=pkg:/web/server/apache-24/module/apache-wsgi-35@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.human-version value=$(HUMAN_VERSION)
 set name=pkg.summary \
```
Comment (mtelka) [inline components/web/apache2-modules/mod_wsgi/mod_wsgi-35.p5m]: Python 3.5 is no longer supported. I suggest to obsolete `apache-wsgi-35`.

**Review by mtelka — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
Thanks for the hint, I changed this.

File: components/web/apache2-modules/mod_wsgi/history [hunk 3]
```diff
@@ -1,2 +1,4 @@
 web/server/apache-24/module/apache-wsgi-34@4.5.22-2020.0.1.3
 web/server/apache-24/module/apache-wsgi-26@3.4,5.11-2016.0.1.0
+web/server/apache-24/module/apache-wsgi-27@4.9.0,5.11-2021.0.0.0
+web/server/apache-24/module/apache-wsgi-35@4.9.0,5.11-2021.0.0.0
```
Comment (AndWac) [inline components/web/apache2-modules/mod_wsgi/history]: When you add an FMRI into a history file make sure to increment its COMPONENT_REVISION.
Thus,
web/server/apache-24/module/apache-wsgi-27@4.9.0,5.11-2021.0.0.1

The same applies for wsgi-35.

**Pull request merged**